test/unit/spec/PasswordReset_spec.js

```diff
@@ -228,7 +230,8 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect,
           password: 'newpwd'
         }, jasmine.any(Function));
         return tick();
-      }).then(function() {
+      })
+      .then(function() {
```

Comment:
This is intentionally **tick()** and not **waitForSpyCall** as **successSpy** will not be called as the save is prevented intentionally.